templates/.gitignore.tpl

```diff
@@ -61,6 +61,13 @@ Pulumi.*.yaml
 # Documentation output
 /apidocs
 
+### Start ignores inserted by other modules
+{{- $extraHook := (stencil.GetModuleHook "extraIgnores") }}
+{{- range $extraHook }}
+{{- .}}
```

Comment:
You'll want the `-` to be removed otherwise, it'll print them all on the same line.

Reply:
that's not true. It worked as expected
<img width="484" alt="image" src="https://user-images.githubusercontent.com/6810045/225383604-863b3ab4-5c19-43a9-b507-a35e800fdc97.png">

Reply:
That's.... odd, `{{-` trips the preceding whitespace, [as per the docs](https://pkg.go.dev/text/template#hdr-Text_and_spaces). If it's working that's cool